log/api_test.go

```diff
@@ -25,6 +25,12 @@ type fieldsTest struct {
 	expected      frozen.Map
 }
 
+type testHook struct { }
```

Comment:
Too late now, but I can see from the formatting of this one line that `go fmt` / `goimports` has not been run over this.

The best thing to do (IMHO) is to configure your editor to run `goimports` on save and never think about it again.

Reply:
Good advice. Thanks.